golang/cosmos/x/swingset/types/sim-params.go

```diff
@@ -35,11 +35,11 @@ var (
 	// Fees are denominated in units of $1 RUN.
 	DefaultFeeUnitPrice = sdk.NewCoins(sdk.NewInt64Coin("urun", 1000000))
 
-	DefaultBeansPerFeeUnit     = sdk.NewUint(1000000000000)                        // $1
-	DefaultBeansPerInboundTx   = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(100000))   // $0.00001
-	DefaultBeansPerMessage     = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(1000000))  // $0.000001
-	DefaultBeansPerMessageByte = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(50000000)) // $0.0000002
-	DefaultBeansPerMinFeeDebit = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(4))        // $0.25
+	DefaultBeansPerFeeUnit     = sdk.NewUint(1000000000000)                     // $1
+	DefaultBeansPerInboundTx   = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(100))   // $0.01
+	DefaultBeansPerMessage     = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(1000))  // $0.001
+	DefaultBeansPerMessageByte = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(50000)) // $0.0002
+	DefaultBeansPerMinFeeDebit = DefaultBeansPerFeeUnit.Quo(sdk.NewUint(5))     // $0.2
```

Comment:
Where do these numbers come from? I suspect there was discussion elsewhere; summarize it in a comment here?

Reply:
> Where do these numbers come from? I suspect there was discussion elsewhere; summarize it in a comment here?

These numbers make deploying an ag-solo cost less than $1.00.  I don't want this PR to have to wait for the cost model and calculation of what the defaults should be, just that the mechanism is available (and can already be tweaked via governance or `genesis.json` parameters).